src/buildHTML.js

```diff
@@ -1491,6 +1491,27 @@ groupTypes.mclass = function(group, options) {
     return makeSpan([group.value.mclass], elements, options);
 };
 
+groupTypes.not = function(group, options, prev) {
+    const inner = buildExpression(group.value.body, options);
+    const solidus = buildCommon.mathsym(
+        "\\not", group.mode, options.getColor(), ["mrel"]);
+
+    // 0.778 = -1 * ptPerEm * -778
+    // -778 is the horizontal correction that has been added to glyph.
+    // TODO(kevinb) update the font to remove this correction so that we don't
+    // have to reverse its effects here.
+    solidus.style.position = 'absolute';
+    const transform = 'translate(0.778em, 0em)';
+    solidus.style.transform = transform;
+    solidus.style.MozTransform = transform;
+    solidus.style.MsTransform = transform;
+    solidus.style.WebkitTransform = transform;
+
+    // The reason for flipping the order is that TeX put the solidus at the
+    // start of a group, not the end.
+    return makeSpan(["mord mrel relpos"], [solidus, ...inner]);
```

Comment:
This commit increases the size of the gzipped file by about 4.5k, and I guess it is most likely due to this command here. Not that I'm objecting – switching to ES6 is of little value unless we actually start using some of its features. But we should be aware of this cost.